mainapp/views.py

```diff
@@ -43,6 +43,163 @@ def home_page(request):
     return render(request, 'mainapp/home.html', context)
 
 
+def food(request):
```

Comment:
Use a single view to render different categories. Use the category passed in URL to decide which data needs be sent to HTML page